Program.cs

```diff
@@ -30,10 +30,10 @@ class TheGame
 
         public void Begin()
         {
-            playerType();
-
             if (isPlayerNew)
             {
+                playerType();
```

Comment:
This adds no value - and was branched off the wrong branch -- you should have started the branch off of Master ( the last approved state ) not a branch that was still under review.